tests/apps/courses/test_models_course.py

```diff
@@ -358,7 +430,7 @@ def test_models_course_get_organizations(self):
             list(course.public_extension.get_organizations()), published_organizations
         )
 
-    def test_models_course_get_organizations_language(self):
+    def test_models_course_get_organizations_language_current(self):
```

Comment:
Not sure that this test is still relevant and its purpose could be misunderstood as the docstring lets think that the page retrieves organizations only in the current language.
From my point of view, `test_models_course_get_organizations_language_fallback` is enough